map.json

```diff
@@ -33,5 +33,10 @@
 		"id" : "win-bcdedit-ignoreAllFailure",
 		"path" : "src/windows/win-ignoreAllFailures.ps1",
 		"description" : "Ignore errors if there is a failed boot, failed shutdown, or failed checkpoint. The computer will attempt to boot normally after an error occurs."
+	},
+	{
+		"id" : "linux-alar-fki",
+		"path" : "src/linux/linux-alar-fki.sh",
+		"description" : "alar-fki allows to recover a failed VM. Three recover scenarios are possible: fstab, initrd and kernel. NOTE: use option --run-on-repair. Se the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md"
```

Comment:
typos: se the docu ? 

also i don't think the first part is needed 'alar-fki allows to recover a failed VM'. Maybe it can be, "alar-fki repairs three scenarios: fstab, initrd, and kernel. NOTE: must use with option --run-on-repair." and the rest.

Reply:
What about
linux-alar-fki provides three recover scenarios: fstab, initrd and kernel. NOTE: use option --run-on-repair. Consult the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md"
	}

Reply:
yeah thats is good.